test/helper.ts

```diff
@@ -17,8 +17,11 @@ export const ANSIBLE_COLLECTIONS_FIXTURES_BASE_PATH = path.resolve(
   "common",
   "collections"
 );
-const LIGHTSPEED_INLINE_SUGGESTION_WAIT_TIME = 10000;
-const LIGHTSPEED_INLINE_SUGGESTION_AFTER_COMMIT_WAIT_TIME = 2000;
+const LIGHTSPEED_ACCESS_TOKEN = process.env.LIGHTSPEED_ACCESS_TOKEN || "dummy";
```

Comment:
When the local mock server is used, we should be able to reduce the wait time for each API call. In this PR, I reduced them to 1/10.